docs/conduit/WritingTxnsToFile.md

```diff
@@ -0,0 +1,191 @@
+[developer portal](https://developer.algorand.org/).
+
+Here is the completed importer config:
+```yaml
+importer:
+    name: algod
+    config:
+      netaddr: "http://127.0.0.1:8080"
+      token: "e36c01fc77e490f23e61899c0c22c6390d0fff1443af2c95d056dc5ce4e61302"
+```
+
+## Setting up our Processor
+
+The processor section in our generated config is empty, so we'll need to fill in that section with the proper data
+for the filter processor. We can paste in the output of our list command for that.
+```yaml
+name: filter_processor
+config:
+  # Filters is a list of boolean expressions that can search the payset transactions.
+  # See README for more info.
+  filters:
+    - any:
+      - tag: txn.rcv
+        expression-type: exact
+        expression: "ADDRESS"
+```
+Since conduit allows us to have multiple processors, we need to ensure that this is part of a list under the processors
+key. The filter processor uses the tag of a property and allows us to specify an exact value to match or a regex.
+For our use case we'll grab the address of a wallet I've created on testnet, `NVCAFYNKJL2NGAIZHWLIKI6HGMTLYXL7BXPBO7NXX4A7GMMWKNFKFKDKP4`.
+
+That should give us exactly what we want, a filter that only allows transaction through for which the receiver is my
+account. However, there is a lot more you can do with the filter processor. To learn more about the possible uses, take
+a look at the individual plugin documentation [here](./plugins/filter_processor.md).
+
+## Setting up our Exporter
+
+For the exporter the setup is simple. I'll set the directory output of my blocks to a temporary directory,
+`block-dir: "/tmp/conduit-blocks/"`.
+
+## Running the pipeline
+Now we should have a fully valid config, so let's try it out. Here's the full config I ended up with
+(with comments removed)
+```yaml
+log-level: "INFO"
+importer:
+    name: algod
+    config:
+      netaddr: "http://127.0.0.1:8080"
+      token: "e36c01fc77e490f23e61899c0c22c6390d0fff1443af2c95d056dc5ce4e61302"
+processors:
+  name: filter_processor
+  config:
+    filters:
+      - any:
+       - tag: txn.rcv
+         expression-type: exact
+         expression: "NVCAFYNKJL2NGAIZHWLIKI6HGMTLYXL7BXPBO7NXX4A7GMMWKNFKFKDKP4"
+exporter:
+    name: file_writer
+    config:
+      block-dir: "/tmp/conduit-blocks/"
+```
+
+There are two things to address before our example becomes useful.
+1. We need to get a payment transaction to our account.
+For me, it's easiest to use the testnet dispenser, so I've done that. You can look at my transaction for yourself,
+block #26141781 on testnet.
+2. Skip rounds
+To avoid having to run algod all the way from genesis to the most recent round, you can use catchpoint catchup to
+fast-forward to a more recent block. Similarly, we want to be able to run Conduit pipelines from whichever round is
+most relevant and useful for us.
+To run conduit from a round other than 0, use the `-r` flag. Here's my command,
```

Comment:
I don't feel strongly about this one. Generally for documentation I usually prefer to use the long form of commands:
```suggestion
To run conduit from a round other than 0, use the `--next-round-override` or `-r` flag. Here's my command,
```